core_brain/shadow_exec.py

```
@@ -0,0 +1,672 @@
+            posted_ts=now_ms, last_polled_ts=now_ms, pair_id=pair_id,
+        ))
+        self._registry.record_fill(FillRecord(
+            trade_id=f"{SHADOW_TRADE_PREFIX}{uuid.uuid4().hex[:16]}",
+            order_uuid=local_id, size=shares, price=price,
+            venue_ts=now_ms, recorded_ts=now_ms,
+        ))
+        return {"success": True, "orderID": local_id, "status": "matched",
+                "price": price, "size": shares}
+
+
+def shadow_positions(registry: OrderRegistry, db_path: Path | str) -> dict[str, float]:
```

Ruff (0.16.2)

[warning] Unused function argument: `registry`

(ARG001)